client/app/scripts/superdesk-publish/views/publish-queue.html

```diff
@@ -24,19 +23,40 @@ <h3 class="page-nav-title" ng-hide="multiSelectCount > 0" translate>Publish Queu
                         </button>
                         <ul class="dropdown-menu">
                             <li>
-                                <a href="" ng-click="filterSchedule(null, 'subscriber')">{{ :: 'None' | translate}}</a>
+                                <a href="" ng-click="filterSchedule(null, 'subscriber')" translate>None</a>
                             </li>
                             <li ng-repeat="subscriber in subscribers track by subscriber._id">
                                 <a href="" ng-click="filterSchedule(subscriber, 'subscriber')">{{ subscriber.name }}</a>
                             </li>
                         </ul>
                     </div>
+                    <span class="lab pull-left" translate>Status:</span>
+                    <div class="dropdown pull-left" dropdown>
+                        <button class="dropdown-toggle" dropdown-toggle> {{ selectedFilterStatus }} <b class="caret"></b>
+                        </button>
+                        <ul class="dropdown-menu">
+                            <li>
+                                <a href="" ng-click="filterStatus(null, 'status')" translate>None</a>
+                            </li>
+                            <li ng-repeat="status in publish_queue_statuses track by $index">
```

Comment:
currently it is fixed as its coming from `$scope.publish_queue_statuses = ['pending', 'in-progress', 'success', 'error'];`. so using track by $index for this instance.